wallet/src/repository/wallet/mod.rs

```diff
@@ -1668,16 +1659,27 @@ where
     }
 
     /// Run a predicate on the state of a wallet in a thread safe manner, thanks to a write lock.
-    pub fn lock_and_update_state<P, O>(&self, predicate: P) -> Result<O>
+    pub fn _lock_and_update_state<P, O>(&self, predicate: P) -> Result<O>
```

Comment:
Add unused attribute, to comply with the existing naming conventions

```suggestion
    #[allow(unused)]
    pub fn lock_and_update_state<P, O>(&self, predicate: P) -> Result<O>
```